docs/rp2/quickref.rst

```diff
@@ -55,6 +55,60 @@ The :mod:`rp2` module::
 
     import rp2
 
+Networking
+----------
+
+WLAN
+^^^^
+
+.. note::
+   This section applies only to devices that include WiFi support, such as the `Pico W`_ and `Pico 2 W`_.
+
+The :class:`network.WLAN` class in the :mod:`network` module::
+
+    import network
+
+    wlan = network.WLAN(network.WLAN.IF_STA) # create station interface
+    wlan.active(True)       # activate the interface
+    wlan.scan()             # scan for access points
+    wlan.isconnected()      # check if the station is connected to an AP
+    wlan.connect('ssid', 'key') # connect to an AP
+    wlan.config('mac')      # get the interface's MAC address
+    wlan.ipconfig('addr4')  # get the interface's IPv4 addresses
+
+    ap = network.WLAN(network.WLAN.IF_AP) # create access-point interface
+    ap.config(ssid='RP2-AP') # set the SSID of the access point
+    ap.config(max_clients=10) # set how many clients can connect to the network
+    ap.active(True)         # activate the interface
```

Comment:
You could also add the line `ap.status("stations")` as an example of how to get a list of connected STAs.

Reply:
OK, but if so I'll also update the ESP32 quickref.

Reply:
I didn't end up making this change because I couldn't find any documentation for the option. Seems like that should be in place first?

Reply:
Documentation for stations was just added now in 4d034f817c861b90cbe7ab0cb59d85036031750f

Do you want to add the example?  Happy either way.

Reply:
We can add this another time.